Source/SuperMario3DWorld/rules.txt

```diff
@@ -110,29 +123,125 @@ $gameWidth = 1280
 $gameHeight = 720
 
 
-# Full Res
+
+
+
 [TextureRedefine]
 width = 1280
 height = 720
-formatsExcluded = 0x008,0x41A,0x034,0x035 # exclude obvious textures
-#formats = 0x01a,0x019,0x80e,0x816
+formatsExcluded = 0x008,0x41A,0x034,0x035 # need to verify 0x008
 overwriteWidth = ($width/$gameWidth) * 1280
-overwriteHeight = ($height/$gameHeight) * 720
+overwriteHeight  = ($height/$gameHeight) * 720
+
+[TextureRedefine] #shadow
+width = 1024
+height = 1024
+formats =  0x005
+overwriteWidth = ($width/$gameWidth) * 1024
+overwriteHeight = ($height/$gameHeight) * 1024
```

Comment:
There's separate shadow packs

Reply:
Yes, XYZ pack lets you scale as you please, by default we scale all and fix all viewports right?